build.sh

```diff
@@ -184,7 +185,7 @@ case $OSName in
 esac
 __MSBuildBuildArch=x64
 __BuildType=Debug
-__CMakeArgs=DEBUG
```

Comment:
Does this mean we have no default cmake DEBUG arg without any parameters passed to build.sh?